tools/publish/BUILD.bazel

```diff
@@ -1,8 +1,15 @@
 load("//python:pip.bzl", "compile_pip_requirements")
+load("//python/entry_points:py_console_script_binary.bzl", "py_console_script_binary")
 
 compile_pip_requirements(
     name = "requirements",
     src = "requirements.in",
     requirements_darwin = "requirements_darwin.txt",
     requirements_windows = "requirements_windows.txt",
 )
+
+py_console_script_binary(
```

Comment:
We should use the version-aware rule to make sure this uses the python version the deps were built for.